services/ui-src/src/forms/mcpar/dgo/dgo.json

```diff
@@ -1,18 +1,66 @@
+    "drawerTitle": "Report on grievances for"
   },
   "form": {
     "id": "dgo",
     "fields": [
       {
         "id": "dgo-1",
-        "type": "text",
+        "type": "number",
+        "props": {
+          "label": "D1.IV.10 Grievances resolved",
+          "hint": "Enter the total number of grievances resolved by the plan during the reporting year.</p>A grievance is \"resolved\" when it has reached completion and been closed by the plan.",
+          "mask": "comma-separated"
+        }
+      },
+      {
+        "id": "dgo-2",
+        "type": "number",
+        "props": {
+          "label": "D1.IV.11 Active grievances",
+          "hint": "Enter the total number of grievances still pending or in process (not yet resolved) as of the first day of the last month of the reporting year.",
+          "mask": "comma-separated"
+        }
+      },
+      {
+        "id": "dgo-3",
+        "type": "number",
+        "props": {
+          "label": "D1.IV.12 Grievances filed on behalf of LTSS users",
+          "hint": "Enter the total number of grievances filed during the reporting year by or on behalf of LTSS users.</p>An LTSS user is an enrollee who received at least one LTSS service at any point during the reporting year (regardless of whether the enrollee was actively receiving LTSS at the time that the grievance was filed).",
+          "mask": "comma-separated"
+        }
+      },
+      {
+        "id": "dgo-4",
+        "type": "number",
+        "props": {
+          "label": "D1.IV.13 Number of critical incidents filed during the reporting period by (or on behalf of) an LTSS user who previously filed a grievance",
+          "hint": "For managed care plans that cover LTSS, enter the number of critical incidents filed within the reporting period by (or on behalf of) LTSS users who previously filed grievances in the reporting year.</p>If the managed care plan does not cover LTSS, enter \"N/A\". Additionally, if the state already submitted this data for the reporting year via the CMS readiness review appeal and grievance report (because the managed care program or plan were new or serving new populations during the reporting year), and the readiness review tool was submitted for at least 6 months of the reporting year, enter “N/A”.</p>The grievance and critical incident do not have to have been \"related\" to the same issue — they only need to have been filed by (or on behalf of) the same enrollee. Neither the critical incident nor the grievance need to have been filed in relation to delivery of LTSS — they may have been filed for any reason, related to any service received (or desired) by an LTSS user.</p>To calculate this number, states or managed care plans should first identify the LTSS users for whom critical incidents were filed during the reporting year, then determine whether those enrollees had filed an appeal during the reporting year, and whether the filing of the appeal preceded the filing of the critical incident.",
```

Comment:
On all these that have multi-paragraph hint text, with only closing `p` tags it renders in the DOM like this:
![Screen Shot 2022-08-26 at 08 54 32](https://user-images.githubusercontent.com/27705928/186919898-d5ebbb50-61f8-427a-80bd-6017bad63466.png)

Suggest either adding an opening `<p>` tag or changing the closing `</p>` tag to a line break instead.